lib/lifecycles/bump.js

```diff
@@ -222,10 +222,16 @@ function updateConfigs(args, newVersion) {
     }
     const configPath = path.resolve(process.cwd(), updater.filename);
     try {
-      if (dotgit.ignore(updater.filename)) return;
+      if (dotgit.ignore(updater.filename)) {
+        console.debug(updater.filename + ' is ignored in Git')
```

Comment:
```suggestion
        console.debug(`Not updating file '${updater.filename}', as it is ignored in Git`)
```

^ I think it's worth being explicit about what is happening in log messages. Also, I like to put the variable contents in quotes, as then empty variables are clearer in logs.